README.md

```diff
@@ -8,6 +8,9 @@ A timer component for React that counts down to zero for a specified number of m
  `initialTimeRemaining: Number`
  The time remaining for the countdown (in ms).
 
+`setClass: String (optional -- default: 'timer')`
```

Comment:
Agreed, I take back the idea of always including a 'timer' class. That's not really the concern of this component. Then I think calling it `className` makes more sense and would make this behave similarly to many other React components.